exchanges/order/order_types.go

```diff
@@ -371,6 +368,25 @@ const (
 	MissingData
 )
 
+// TimeInForce enforces a standard for time-in-force values across the code base.
+type TimeInForce uint8
+
+// TimeInForce types
+const (
+	UnknownTIF  TimeInForce = 0
+	GTC         TimeInForce = iota // GTC represents GoodTillCancel
+	GTT                            // GTT represents GoodTillTime
+	FOK                            // FOK represents FillOrKill
+	IOC                            // IOC represents ImmediateOrCancel
+	PostOnlyGTC                    // PostOnlyGCT represents PostOnlyGoodTilCancelled
```

Comment:
Can we RM post only bool now if we are going to use this?

Reply:
I don't think so. We can use this TIF value to set GCT and Post only but this one can not replace the PostOnly field